test_daily_workflow.py

```
@@ -1,11 +1,14 @@
 from __future__ import annotations
+import osqp_preimport
```

Flake8 (7.3.0)

[error] 'osqp_preimport' imported but unused

(F401)